src/repl.zig

```diff
@@ -29,6 +29,7 @@ pub const Parser = struct {
         BadIdentifier,
         MissingEqualBetweenKeyValuePair,
         NoSyntaxMatch,
+        UnsupportedSliceOperation,
```

Comment:
```suggestion
        SliceOperationUnsupported,
```

Per TigerStyle, to put the noun first and the qualifier last. This way, if `SliceOperation` has other errors, they all share the same prefix.

Note also that other errors here, before this PR, don't follow TigerStyle. We could go through these and probably clean up the naming.